plugins/aws/fix_plugin_aws/collector.py

```diff
@@ -252,6 +253,10 @@ def get_last_run() -> Optional[datetime]:
                     log.warning(f"Unexpected node type {node} in graph")
                     raise Exception("Only AWS resources expected")
 
+            # add access edges
+            access_edge_creator = AccessEdgeCreator(global_builder)
+            access_edge_creator.add_access_edges()
```

Comment:
I assume that no API calls are required at that moment in time, since all data is available?

Reply:
yes